src/verifiers/documentStatus/didSigned/didSignedDocumentStatus.ts

```diff
@@ -182,16 +189,23 @@ const verifyV3 = async (
 
   const issuedOnAll = verificationResult.verified;
 
-  const getRevocationStatus = async (docType: v3.RevocationType, location: string) => {
+  const getRevocationStatus = async (docType: v3.RevocationType, location: string | undefined) => {
```

Comment:
```suggestion
  const getRevocationStatus = async (docType: v3.RevocationType, location?: string) => {
```

Hmm i think we use `?` most of the time right?